src/main/java/com/pingcap/tikv/codec/LongUtils.java

```diff
@@ -65,7 +67,7 @@ public static void writeULongFull(CodecDataOutput cdo, long lVal, boolean compar
      * @param lVal The data to encode
      */
     public static void writeLong(CodecDataOutput cdo, long lVal) {
-        cdo.writeLong(CodecUtil.flipSignBit(lVal));
+        cdo.writeLong(TableCodec.flipSignBit(lVal));
```

Comment:
Why flipSignBit needs to be in  here? A little bit odd to refactor like this TBH. It might be better stay in Long codec related classes if proved nowhere else use it.